source/geometries/Next100EnergyPlane.cc

```diff
@@ -98,154 +98,114 @@ namespace nexus {
   }
 
 
-  void Next100EnergyPlane::SetMotherLogicalVolume(G4LogicalVolume* mother_logic)
-  {
-    mother_logic_ = mother_logic;
-  }
-
-
-  void Next100EnergyPlane::SetSapphireSurfaceZPos(G4double z)
+  void Next100EnergyPlane::Construct()
   {
-    end_of_sapphire_posz_ = z;
-  }
 
+    G4double offset = 1. * cm;
 
-  void Next100EnergyPlane::Construct()
-  {
     GeneratePositions();
 
     /// Copper Plate ///
-    G4Tubs* copper_plate_origin_solid =
-      new G4Tubs("COPPER_PLATE_ORIGIN", 0., copper_plate_diam_/2.,
-		 copper_plate_thickn_/2., 0., twopi);
+    G4Tubs* copper_plate_origin_solid =  new G4Tubs("COPPER_PLATE_ORIGIN", 0., copper_plate_diam_/2.,
+                                                    copper_plate_thickn_/2., 0., twopi);
 
-    G4double offset = 1. * cm;
-
-    /// Holes for gas flow ///
+    /// Hole for gas flow ///
     G4Tubs* copper_plate_gas_hole_solid =
       new G4Tubs("COPPER_PLATE_CENTRAL_HOLE", 0., gas_hole_diam_/2.,
-		 (copper_plate_thickn_ + offset)/2., 0., twopi);
+		             (copper_plate_thickn_ + offset)/2., 0., twopi);
```

Comment:
The hole doesn't reach the back of the copper plate (it is shorter, about 103mm according to the 2D drawings).